### DISCUSSION TITLE
Add an option for CSV

### DISCUSSION THREAD

**Discussion by kbostick88** [mza921/Plex-Auto-Collections#179] [Ideas]
Can you add an option to add a CSV option. If we had a list of movies/shows in a CSV file, allow that file to be used to create the collection

**Comment by JettLagZulu** [comment]
> Can you add an option to add a CSV option. If we had a list of movies/shows in a CSV file, allow that file to be used to create the collection

Second for this. This would provide a huge increase in functionality. Provide a template and we can dump lists out of any multitude of sources (icheckmovies, etc).

**Comment by burkasaurusrex** [comment]
I'm a bit hesitant to introduce CSVs on top of the YAML?

You should be able to accomplish this with the individual [TMDb movie](https://github.com/mza921/Plex-Auto-Collections#tmdb-movie-list-type) or [TMDb show](https://github.com/mza921/Plex-Auto-Collections#tmdb-show-list-type) or [TVDB show](https://github.com/mza921/Plex-Auto-Collections#tvdb-show-list-type). Something like:

```yaml
collections:
  - My Random Collection:
    - tmdb_movie:
      - id#1
      - id#2
      - id#3
```

The list of IDs could be infinitely long, you'd just have to reformat from whatever source. Not sure if that makes sense?

**Comment by kbostick88** [comment]
That makes sense but would be very complicated to automate. If it accepted
a csv file you could pull collections from a site/source that is constantly
updated to maintain the collection. For example a collection for movies on
Netflix, these would not all be from the studio netflix. Many sites list
all the movies/shows on netflix and create a csv list of them.. it would be
a ton of work to try and change the format and automate that process unless
i am missing something

**Reply by meisnate12** [reply]
Can you provide a link to a csv that's constantly updated and then maybe it and more like it could be added but we'd need an example
